### ISSUE TITLE
Update Account Creation Screen to Use Full Name Instead of First and Last Name

### ISSUE THREAD

**Issue by TheBoatyMcBoatFace** [OpenATO/User-Interface#1]
### Overview

The current account creation screen asks for a user's first and last name, but this can be problematic and exclusive as not all individuals have standard western European names. The goal of this issue is to update the screen to use the full name instead.

**Note:** The current Postgres schema requires `First Name` & `Last Name`. This needs to be addressed.

This improvement brought to light the need for an 'Improvement' issue form. The creation of that form is tracked at 
- https://github.com/OpenATO/.github/issues/1

### Background

The use of full name as the sole identifier in forms is the industry best practice, as demonstrated in the referenced article. This change will make the account creation process more inclusive and user-friendly.

**🤠 Hat Tip:** @TheETupper for bringing this to light in [Slack](https://civicactions.slack.com/archives/C01PXBRSL1Y/p1675351250895829?thread_ts=1675311184.441199&cid=C01PXBRSL1Y)

#### [Why Your Form Only Needs One Name Field](https://uxmovement.com/forms/why-your-form-only-needs-one-name-field/)
> Does your form have two separate fields for the user's name? If it does, you're making it harder for them to fill out your form. But there's a better way. Split Name Fields Cause Problems Not every user has a first and last name. Their cultural background determines how their name is formed. For example, […]

### Acceptance Criteria

- [ ] The account creation screen should only ask for a user's full name
- [ ] The full name should be stored and used as the identifier for the user in the system
- [ ] The change should not impact any existing functionality or data storage systems
- [ ] The updated screen should be visually appealing and easy to use for all users
- [ ] Any necessary documentation and code changes should be made and committed to the appropriate OpenATO repo(s)

### Team

🖼️ Front End
